package.json

```diff
@@ -79,6 +85,7 @@
     "istanbul": "^0.4.0",
     "mocha": "^3.0.0",
     "mockery": "^2.0.0",
+    "node-sass": "^4.5.3",
```

Comment:
It's a requirment that you have node-sass as a dependency in your app if mincer is to handle .scss files:https://github.com/nodeca/mincer#styling-with-sass